rasa/core/processor.py

```diff
@@ -264,23 +266,32 @@ def _has_message_after_reminder(
         ):
             logger.debug(
                 "Canceled reminder because it is outdated. "
-                "(event: {} id: {})".format(
-                    reminder_event.action_name, reminder_event.name
-                )
+                "(event: {} id: {})".format(reminder_event.intent, reminder_event.name)
```

Comment:
We should use f-strings here instead:
```python
logger.debug(
                f"Canceled reminder because it is outdated. "
                f"(event: {reminder_event.intent} id: {reminder_event.name})"
)
```